packages/inference/src/index.ts

```diff
@@ -3,5 +3,8 @@ export { InferenceOutputError } from "./lib/InferenceOutputError";
 export * from "./types";
 export * from "./tasks";
 
-import * as snippets from "./snippets/index.js";
+let snippets = {};
+if (typeof window === "undefined") {
+	snippets = import("./snippets").then((mod) => mod.default);
+}
 export { snippets };
```

Comment:
@coyotte508 what would be the best way to do a conditional import here? I haven't found any convincing solution. In `@huggingface/hub` it's not really a problem as everything is included using `export * from "./lib";` (here).

Reply:
I'm not sure there's an issue? Is there an error when just keeping the code as before?

Reply:
it felt weird to do `import * as snippets from "./snippets/index.js";` when we know `./snippets/index.js` might not be in the final package. But looks like tooling is done the correct way so perfect :smile: